addons/mail/static/src/components/messaging_menu/messaging_menu.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.MessagingMenu" owl="1">
         <div class="o_MessagingMenu dropdown" t-att-class="{ 'show': messagingMenu ? messagingMenu.isOpen : false, 'o-mobile': messaging ? messaging.device.isMobile : false }" t-attf-class="{{ className }}" t-ref="root">
             <a class="o_MessagingMenu_toggler dropdown-toggle o-no-caret o-dropdown--narrow" t-att-class="{ 'o-no-notification': messagingMenu ? !messagingMenu.counter : false }" href="#" title="Conversations" role="button" t-att-aria-expanded="messagingMenu and messagingMenu.isOpen ? 'true' : 'false'" aria-haspopup="true" t-on-click="_onClickToggler">
-                <i class="o_MessagingMenu_icon fa fa-comments" role="img" aria-label="Messages"/>
+                <i class="o_MessagingMenu_icon oi oi-forum" role="img" aria-label="Messages"/>
```

Comment:
Change looks unrelated to pager, wrong PR split?
(but ok for the change of icon in general)

Reply:
@Brieuc-brd has cherry-pick my commit with the new icons.
Icons will be  be merged either here or in the navbar branch https://github.com/odoo/odoo/pull/82434 (that PR it's taking longer than expected)